src/test.coffee

```diff
@@ -28,7 +25,8 @@ class Test
     true
 
   schemaVersion: ->
-    if @schema?.contains('$schema')
+    # JSON-schema in RAML are always objects so..
+    if /^[^{]*\{[\s\S]*\}[^}]*/.test(@schema)
```

Comment:
It will test for a JSON-like object, because a empty object `{}` is valid JSON too.

CSON-Schema will not pass that (?) since braces are not mandatory for the root schema.

I've working on some RAML projects that are using JSON-Schema but missing the `$schema` and `id` property within and works perfectly out of the box.

Further we can use `{ "type": "string" }` as schema and it's perfectly valid for some tools.

This also avoids false positives evaluating empty objects `{}` as CSON-Schema.